vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/PositionCachingTouchAction.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2019-2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.mobileapp.action;
+
+import io.appium.java_client.PerformsTouchActions;
+import io.appium.java_client.TouchAction;
+import io.appium.java_client.touch.offset.PointOption;
+
+@SuppressWarnings("rawtypes")
+public class PositionCachingTouchAction extends TouchAction<PositionCachingTouchAction>
```

Comment:
https://github.com/appium/java-client/commit/a1f802e553b916902c7fc1ddb6ba77063a54f78d - we'll be in trouble soon